libvuln/opts.go

```diff
@@ -59,6 +60,11 @@ type Opts struct {
 	//
 	// This list will me merged with the default matchers.
 	Matchers []driver.Matcher
+	// Configs is a map of functions for configuration of Updaters.
+	Configs map[string]driver.ConfigUnmarshaler
```

Comment:
May as well call this UpdaterConfigs if we ever need to pass Matcher configs down the line.